### PR TITLE
Add contributing documentation

### PR DESCRIPTION
**Pull request by joshuaboniface**
Adds documentation on contributing to Jellyfin.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+We adhere to a fairly standard fork-and-PR model of development:
+
+1. Fork a copy of the Jellyfin repository to your own user.
+
+2. Make your changes on a local feature branch of your copy of the repository.
+
+3. Submit a pull request from your feature branch back to the upstream master branch.
+
+All PRs require review before merging. Once that is done, if the PR is not tagged with the WIP label, it should be immediately merged by the reviewer assuming that they have write access. If you do not have write access, please eel free to leave reviews as well and suggest changes as they will be taken into account as well.
```
Comment (anthonylavado) [inline CONTRIBUTING.md]: "eel" to "feel"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Developer Access
+
+We are open with granting developer (merge/write) access to those who show interest in contributing and reviewing PRs. Such access can be granted by a current developer. Unruly or abusive developers can be removed at any time by the core team, so please take your responsibility seriously. We're all in this together trying to make the best free software media platform we can.
+
+## How can I contribute?
+
+If you're a .NET C# or Javascript developer, please feel free to jump right. If you're not a developer, please consider helping out by contributing documentation, testing the latest releases and reporting bugs, or by telling your friends about the project! Any little bit helps.
```
Comment (anthonylavado) [inline CONTRIBUTING.md]: "jump right" to "jump right in".

Move the "If you're not a developer..." to a new line?

File: README.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ While our first priority is a stable build, we will eventually add features that
 
 [Feature Requests](http://feathub.com/jellyfin/jellyfin)
 
+## Contributing to Jellyfin
+
+Please see our contributing documentation at https://github.com/jellyfin/jellyfin/blob/master/CONTRIBUTING.md
```
Comment (anthonylavado) [inline README.md]: Make it a nice link in Markdown? I also don't remember how to reference the latest version of that at all times...

Comment (anthonylavado) [reply README.md]: Oh I guess it is just blob/master. Okay, just a nicer link then.

"Please read CONTRIBUTING.md for details on how to submit pull requests to us."

File: CONTRIBUTING.md (hunk 1 above)
Comment (joshuaboniface) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (joshuaboniface) [reply CONTRIBUTING.md]: Fixed and added some more words.

File: README.md (hunk 3 above)
Comment (joshuaboniface) [reply README.md]: Same idea but different wording. Fixed.

**Pull request merged**